lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    valid_gamma = gamma[~np.isnan(gamma)]
</pre><p>Maybe we should explain this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Sure, I'll add a comment